jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -41,6 +41,8 @@ val ENUM_TABLES =
                 EnumTable("facility_connection_states", "facilities\\.connection_state_id"),
                 EnumTable("facility_types", "facilities\\.type_id"),
                 EnumTable("growth_forms", listOf("growth_forms\\.id", ".*\\.growth_form_id")),
+                EnumTable("managed_facility_types", listOf("managed_facility_types\\.id",
```

Comment:
Is "managed facility type" an engineering invention, or is it what the ops team uses for this? I find it a bit confusing in that we already have a concept of a "facility type" in the system that has nothing to do with this, and also there are types here (planting sites) that aren't facilities in the sense we use that word elsewhere in the system.

Reply:
This is what I am trying to capture with the 'managed facility type'
```
“Does your organization manage any of the following? Check all that apply.”

Options are checkboxes:
- Seed Bank/Seed Collection
- Nursery
- Planting Site
```
The value is only captured for downstream analytics, will not be re-rendered in the client (as of current plans).
I could also make it a comma separated value of 'TEXT' type.

Reply:
The word "facility" is my main concern here because it has a specific meaning in our software. The structure you have here with an enum table is totally fine.

How about "managed location types" or "managed operational areas" or something like that?

Reply:
> The word "facility" is my main concern here because it has a specific meaning in our software. The structure you have here with an enum table is totally fine.
> 
> How about "managed location types" or "managed operational areas" or something like that?

Location types sounds good to me. Will use that, thanks so much!

Reply:
> > The word "facility" is my main concern here because it has a specific meaning in our software. The structure you have here with an enum table is totally fine.
> > How about "managed location types" or "managed operational areas" or something like that?
> 
> Location types sounds good to me. Will use that, thanks so much!

Saw your comment in the JIRA, thanks.  Will go with 'location' for now and update later if there's any feedback in the JIRA.